Buildings/Experimental/OpenBuildingControl/ASHRAE/G36/Atomic/EconEnableDisableMultiZone.mo

```diff
@@ -0,0 +1,348 @@
+  CDL.Logical.GreaterEqual greEqu "Logical greater or equal block"
+    annotation (Placement(transformation(extent={{-70,-110},{-50,-90}})));
+  CDL.Logical.Timer timer "Timer gets started as the economizer gets disabled"
+    annotation (Placement(transformation(extent={{30,-70},{50,-50}})));
+  CDL.Logical.Nor nor1 "Logical nor"
+    annotation (Placement(transformation(extent={{-40,200},{-20,220}})));
+  CDL.Logical.Not not2 "Logical \"not\" starts the timer at disable signal "
+    annotation (Placement(transformation(extent={{-10,-70},{10,-50}})));
+  CDL.Logical.Less les1 "Check if the RA damper should be fully open"
+    annotation (Placement(transformation(extent={{-8,-190},{12,-170}})));
+  CDL.Logical.LessEqualThreshold equ(final threshold=Constants.FreezeProtectionStages.stage0)
+    "Logical block to check if the freeze protection is deactivated"
+    annotation (Placement(transformation(extent={{-120,40},{-100,60}})));
+  CDL.Logical.GreaterThreshold greThr(final threshold=Constants.ZoneStates.heating)
+    "Check if zone state is other than heating"
+    annotation (Placement(transformation(extent={{-120,-20},{-100,0}})));
+  CDL.Logical.GreaterThreshold greThr2(final threshold=0) "Check if the timer got started"
+    annotation (Placement(transformation(extent={{88,-182},{108,-162}})));
+  CDL.Logical.And3 and2 "Logical and"
+    annotation (Placement(transformation(extent={{130,-200},{150,-180}})));
+  CDL.Logical.And and1 "Logical \"and\" checks supply fan status"
+    annotation (Placement(transformation(extent={{0,100},{20,120}})));
+  CDL.Conversions.IntegerToReal intToRea "Integer to real converter"
+    annotation (Placement(transformation(extent={{-160,40},{-140,60}})));
+  CDL.Conversions.IntegerToReal intToRea1 "Integer to real converter"
+    annotation (Placement(transformation(extent={{-160,-20},{-140,0}})));
+  CDL.Logical.And and3 "Logical \"and\" checks supply fan status"
+    annotation (Placement(transformation(extent={{-20,-120},{0,-100}})));
+
+equation
+  connect(OutDamSwitch.y, yOutDamPosMax) annotation (Line(points={{61,-140},{61,-140},{190,-140}}, color={0,0,127}));
+  connect(TOut, add1.u1) annotation (Line(points={{-200,270},{-160,270},{-160,256},{-142,256}},
+        color={0,0,127}));
+  connect(TOutCut, add1.u2) annotation (Line(points={{-200,230},{-160,230},{-160,244},{-142,244}},
+        color={0,0,127}));
+  connect(add1.y, hysOutTem.u) annotation (Line(points={{-119,250},{-102,250}}, color={0,0,127}));
+  connect(hOut, add2.u1) annotation (Line(points={{-200,190},{-160,190},{-160,176},{-142,176}},
+        color={0,0,127}));
+  connect(hOutCut, add2.u2)
+    annotation (Line(points={{-200,150},{-160,150},{-160,164},{-142,164}}, color={0,0,127}));
+  connect(add2.y, hysOutEnt.u) annotation (Line(points={{-119,170},{-102,170}}, color={0,0,127}));
+  connect(hysOutTem.y, nor1.u1) annotation (Line(points={{-79,250},{-60,250},{-60,210},{-42,210}},
+    color={255,0,255}));
+  if use_enthalpy then
```

Comment:
this is not allowed in CDL. Only connect statements are allowed.
However, if `hysOutEnt` is removed based on the parameter value, then the connect statement is also removed. Hence, make sure that either `hysOutEnt` or `entSubst` is removed.